crates/napi/src/lib.rs

```diff
@@ -30,6 +30,7 @@ DEALINGS IN THE SOFTWARE.
 //#![deny(clippy::all)]
 #![feature(arbitrary_self_types)]
 #![feature(arbitrary_self_types_pointers)]
+#![feature(ptr_metadata)]
```

Comment:
Do we need this change?

Reply:
Nvm, I assume it's required, considering the other changes.

Reply:
yeah pretty annoying to add this everywhere.  afaict it is needed everywhere there is a turbo task value implementating a trait.  so i just put it every where an error was triggered.  I can double check this one.